cypress/support/index.js

```diff
@@ -62,3 +62,48 @@ before(() => {
 });
 
 Cypress.Screenshot.defaults({ screenshotOnRunFailure: DEBUG_FLAG });
+
+// Configure custom commands eyes-cypress
+import '@applitools/eyes-cypress/commands'
+
+const {
+  Before,
+  After
+} = require("cypress-cucumber-preprocessor/steps");
+
+
+Before({ tags: "@applitools" }, () => {
+  // debugger
+
+  cy.eyesOpen({
+    appName: 'Carbon Sage',
```

Comment:
please create `applitools.config.js` in the same location where we have `cypress.json` file and put all additional setup there using json format as it is mentioned [here](https://www.npmjs.com/package/@applitools/eyes-cypress#configure-plugin-and-commands) 
```
module.exports = {
  appName: 'My app',
  showLogs: true,
  batchName: 'My batch'
  ...
  // all other configuration variables apply
}